src/containers/App/App.js

```diff
@@ -19,27 +19,35 @@ class App extends Component {
   }
 
   render() {
-    return (
-      <div className="App">
-        <Header />
-        <Switch>
-          <Route exact path='/' component={NoteArea} />
-          <Route path='/new-note' render={() => {
-            const note = { title: '', id: shortID.generate(), timestamp: Date.now() }
-            return <NoteForm note={note} items={[]} isEdit={false}/>
-          }} />
-          <Route path='/notes/:id' render={({ match }) => {
-            const { id } = match.params
-            const note = this.props.notes.find(note => note.id === id)
-            const items = this.props.items.filter(item => item.noteID === id)
-            if (note) {
-              return <NoteForm note={note} items={items} isEdit={true}/>
-            }
-          }} />
-          <Route component={NotFound} />
-        </Switch>
-      </div>
-    );
+    const {error} = this.props
+    console.log('app error', this.props.error)
+    switch (error) {
+      case '':
+        return (
+          <div className="App">
+            <Header />
+            <Switch>
+              <Route exact path='/' component={NoteArea} />
+              <Route path='/new-note' render={() => {
+                const note = { title: '', id: shortID.generate(), timestamp: Date.now() }
+                return <NoteForm note={note} items={[]} isEdit={false} />
+              }} />
+              <Route path='/notes/:id' render={({ match }) => {
+                const { id } = match.params
+                const note = this.props.notes.find(note => note.id === id)
+                const items = this.props.items.filter(item => item.noteID === id)
+                if (note) {
+                  return <NoteForm note={note} items={items} isEdit={true} />
+                }
+              }} />
+              <Route component={NotFound} />
+            </Switch>
+          </div>
+        );
+      default:
+        return <div>Error: {error}</div>
```

Comment:
OK this makes sense - if there is an error we will get the display it to the user? Maybe we can make a component to display instead if there is some issue.

Reply:
What about simply throwing an if conditional in there at the top...do we need a switch for the whole thing?